tests/unit/migrations/pending-networking-actions.sql

```diff
@@ -599,7 +599,7 @@ INSERT INTO network_projects (project_id, network_id) VALUES (1, 1);
 -- Data for Name: networking_action; Type: TABLE DATA; Schema: public; Owner: hil
 --
 
-INSERT INTO networking_action (id, nic_id, new_network_id, channel) VALUES (1, 1, 1, 'null');
+INSERT INTO networking_action (id, nic_id, new_network_id, channel) VALUES (2, 1, 1, 'null');
```

Comment:
don't know how I feel about this change, but I can't think of any other solution right now. @zenhack any ideas on this?

Reply:
I am *very* uncomfortable with this.

I also kindof don't like the fact that we're caling one of the request handlers from another. It feels wrong.

Adding the config option would solve needing to modify the database dump, but I don't think that justifies it by itself.

I'm not totally sure what I think the right thing to do here is. Maybe we're losing the plot a little bit here -- The HIL server is supposed to be bare-bones. It's not intended to be a high-level tool. One approach is to just build separate tools for admins to use that do this correctly; we don't actually need server-side support I don't think.

Still waffling a bit. Thoughts?

Reply:
We should do this in the CLI then. Perhaps, when we have a new CLI, we could do `port_connect_nic --clean` which would make 2 calls, one for `port_connect` and another for `revert_port`. Or maybe get rid of the flag and make it the default behavior. That way we don't touch the APIs

Reply:
OK. do you want to just close this then?